doc/dev/release-process.rst

```diff
@@ -205,14 +208,27 @@ See `the Ceph Tracker wiki page that explains how to write the release notes <ht
 
       sync-push octopus
 
+This leaves the packages in a password-protected prerelease area
+at https://download.ceph.com/prerelease.  Verify them from there.  
+When done and ready for release, mv the directories to the parent
+directory.  
+
+
 5. Build Containers
 ===================
 
-Start the following two jobs:
+Prerelease containers (x86 only) are built by 
+https://2.jenkins.ceph.com/job/ceph-container-prerelease-build; run it 
+with appropriate parameters.  Test container images will appear on
+quay.ceph.io in the ceph/prerelease repo.  When satisfied with them,
+start the following two jobs:
 
```

Comment:
I think we can start building the final containers after the `mv * ../` step

Reply:
I suspect that you mean that we cannot start building the final containers *until* the mv step, and that's right and I'll add it

Reply:
Yes thx